modules/backup/main.tf

```diff
@@ -110,6 +110,7 @@ resource "google_workflows_workflow" "sql_export" {
     gcsBucket           = var.export_uri
     dbType              = split("_", data.google_sql_database_instance.backup_instance.database_version)[0]
     compressExport      = var.compress_export
+    databaseNameLog     = var.enable_database_name_log_export
```

Comment:
Hows `logDbName` for this var?

Reply:
Yes! sounds better and I believe it is clearer to understand